components/brave_today/browser/pref_utils.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
This is the beginning of a proposed refactor to locate all the prefs-related code in the same place. I started with channels since I needed to add a test which added a channel subscription.

Reply:
Seems sensible to me.